js/theme/viewmodel.js

```diff
@@ -114,7 +122,10 @@ function renderPosts(api_response, opts) {
  * @param {object} api_response - liveblog API response JSON.
  */
 function updateViewModel(api_response, opts) {
-  vm._items.push.apply(vm._items, api_response._items);
+  if (opts.sort === 'oldest_first')
```

Comment:
Sticking with the Crockford way of doing things, we should use curly braces here (like you did elsewhere: https://github.com/liveblog/liveblog-default-theme/blob/67e6dedcee27b66db0743d9991af397080ffac6f/js/theme/helpers.js#L60).